packages/client/src/service/validate/validate.ts

```diff
@@ -78,6 +95,25 @@ class Validate {
     }
   }
 
+  getErrorMessage(name: string) {
+    switch (name) {
+      case InputId.EMAIL:
+       return ErrorMessages.EMAIL;
+      case InputId.LOGIN:
+        return ErrorMessages.LOGIN;
+      case InputId.FIRST_NAME:
+        return ErrorMessages.FIRST_NAME;
+      case InputId.SECOND_NAME:
+        return ErrorMessages.SECOND_NAME;
+      case InputId.PHONE:
+        return ErrorMessages.PHONE;
+      case InputId.PASSWORD:
+        return ErrorMessages.PASSWORD;
+      default:
+        return "";
+    }
+  }
```

Comment:
[Предложение]
Можно сделать объект. Меньше строк занимает, хорошо читается. И `name` судя по функци - `InputId` по типу
```suggestion
  const mapErrorMessage: Record<InputId, string> = {
    [InputId.EMAIL]: ErrorMessages.EMAIL,
    [InputId.LOGIN]: ErrorMessages.LOGIN,
    [InputId.FIRST_NAME]: ErrorMessages.FIRST_NAME,
    [InputId.SECOND_NAME]: ErrorMessages.SECOND_NAME,
    [InputId.PHONE]: ErrorMessages.PHONE,
    [InputId.PASSWORD]: ErrorMessages.PASSWORD,
  }
```